docker/1.3.1/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    curl \
+    git \
+    wget \
+    vim \
+    jq \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    build-essential \
+    zlib1g-dev \
+    libglib2.0-0 \
+    libgl1-mesa-glx \
+    openjdk-8-jdk-headless
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
```

Comment:
Move all ARGs and ENVs to the top. ARGs before ENV. Open MP envs are an exception.

Reply:
done